### ISSUE TITLE
Add Refresh Button (To Sync With Jira or Github.. etc)

### ISSUE THREAD

**Issue by ErnestUTN** [johannesjo/super-productivity#190]
Greetings

I was hoping that the  button that used to refresh the remote tasks to backlog could be brought back since it was useful to force the update sometimes.

Regards!

#31

**Comment by johannesjo** [comment]
Yes you're right. I having a bit trouble of figuring out the right ui for this. but I will implement it at some point.

**Comment by johannesjo** [comment]
I would suggest just putting this in the project context menu (the three dots in the left-hand menu and when hovering over the project title).

**Comment by vctls** [comment]
Was this command ever reintroduced? I don't see it anywhere.  
I had to disable automatic Jira updates because I was getting rate-limited and couldn't access the Jira web UI.
